src/test/java/com/amadeus/NamespaceTest.java

```diff
@@ -277,6 +280,15 @@ public void testGetMethods() throws ResponseException {
             .thenReturn(multiResponse);
     HotelSentiments hotelSentiments = new HotelSentiments(client);
     TestCase.assertNotNull(hotelSentiments.get(params));
+
+    // Test flight delay predictions
+    Mockito.when(client.get("/v1/travel/predictions/flight-delay", null))
+        .thenReturn(multiResponse);
+    Mockito.when(client.get("/v1/travel/predictions/flight-delay", params))
+        .thenReturn(multiResponse);
+    FlightDelay flightDelay = new FlightDelay(client);
+    TestCase.assertNotNull(flightDelay.get());
+    TestCase.assertNotNull(flightDelay.get(params));
```

Comment:
In the other unit test we check that the response's length is 2... I dont really remember why and if it is very useful...     TestCase.assertEquals(flightOffersPrediction.post().length, 2);

Reply:
At first I had this check, but I removed it because I didn't know if it was relevant to this test case.

Reply:
to be honest, I am not sure either...